test/com/google/cose/Encrypt0MessageTest.java

```diff
@@ -67,4 +67,34 @@ public void testSerializeWithProtectedHeaders() throws CoseException, CborExcept
         + "DD25867374B24BEE54AA5D797C8DC845929ACAA47EF",
         TestUtilities.bytesToHexString(message.serialize()));
   }
+
+  @Test
+  public void testParsingNullCiphertext() throws CborException, CoseException {
+    String cborString = "8340A20101054C02D1F7E6F26C43D4868D87CEF6";
+    Encrypt0Message.deserialize(TestUtilities.hexStringToByteArray(cborString));
+  }
+
+  @Test
```

Comment:
super nitpick: should be separate testcases, so the method name can describe _why_ the builder failure is expected.